popSimManu.tex

```diff
@@ -423,13 +445,15 @@ \subsection*{Use case: comparing methods of demographic inference}
 
 % JK here --- I think it would really help if we described what the models
 % were, briefly, rather than saying who inferred them.
+% ALG: agreed.
 In Figure \ref{fig:n_t_ragsdale} we present results from our $N(t)$ analysis pipeline
-run on whole genome simulations under under a sophisticated demographic model
-and empirical genetic map and a model of human migration out of Africa
-that includes archaic admiture \citep{ragsdale2019models}. In each column of this figure
+run on human whole genome simulations under a sophisticated demographic model
+and empirical genetic map and a model of migration out of Africa
+that includes archaic admiture \citep{ragsdale2019models}. In each row of this figure
 we show inferred $N(t)$ for the three extant populations in the model.
-In each row we show comparisons among the methods (note that we compare two differing
+In each columns we show comparisons among the methods (note that we compare two differing
 sample sizes for \MSMC).
+% ALG: I think column and row were reversed in the text description. Or am I crazy?
```

Comment:
no the way it is written currently is correct...

Reply:
I see. I interpreted that sentence as: Each row is a different method, and each column is a different population. I'll leave as it was, but I think the sentence is kinda confusing.